planet/src/main/webapp/js/about-section-renderer.js

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export const AboutSectionRenderer = {
```

Comment:
Could you elaborate on why we export an object with a function? It seems at first glance like we could export the function directly.

Reply:
Actually, I declared the object like this because it allows me to export more than just one function. I can also export variables if I put then inside the object. It's true that I only have one function in this case but the module is still not complete and I am expecting to add more to it later.

Reply:
Interesting approach. Sounds reasonable.